CHANGELOG.md

```diff
@@ -2,10 +2,13 @@
 
 ## master / unreleased
 
+
+## 0.6.0 / 2020-01-22
+
 Note that the ruler flags need to be changed in this upgrade. You're moving from a single node ruler to something that might need to be sharded.
 If you are running with a high `-ruler.num-workers` and if you're not able to execute all your rules in `-ruler.evaluation-interval`, then you'll need to shard.
 Further, if you're using the configs service, we've upgraded the migration library and this requires some manual intervention. See full
-instructions below to upgrade your Postgres.
+instructions below to upgrade your PostgreSQL.
 
 * [CHANGE] Remove unnecessary configs/flags from the ruler ring config to align with the pattern used in the distributor ring. #1987
```

Comment:
Release 0.4 didn't have a ruler ring, so these particular differences don't need to be listed as a CHANGE.
It would be good to mention the ring alongside the "transition to upstream Prometheus rules manager"

Reply:
What's about now?